File: Unit02/ArrayUtils.java
import java.util.Random;

public class ArrayUtils {
    private static Random RNG = new Random();

    public static int random(int min, int max) {
        int upperBound = max - min + 1;
        int rand = RNG.nextInt(upperBound);
        return rand + min;
    }

    public static int[] randomArray(int size, int min, int max) {
        int[] random = new int[size];

        for(int index=0; index<random.length; index++) {
            random[index] = random(min, max);
        }

        return random;
    }
}

File: Unit02/array_utils.py
import random
import arrays

def random_array (size, min_value, max_value):
    an_array = arrays.Array (size)
    
    index = 0
    while index < size:
        an_array[index] = random.randint (min_value, max_value)
        index += 1

    return an_array

def main ():
    a = random_array (10, 50, 100)
    print (a)

    b = random_array (5, 1000, 10000)
    print (b)

if __name__ == "__main__":
    main ()